src/server/StudioManagementSystem.Core/Migrations/20230507152251_Update_GroupApplications_WithOneToManyRelationship.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace StudioManagementSystem.Core.Migrations
+{
+    /// <inheritdoc />
+    public partial class Update_GroupApplications_WithOneToManyRelationship : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // re-add the table without the unique Fk constraint
+            migrationBuilder.DropTable(name: "GroupApplication");
```

Comment:
to fix an incorrect unique FK, we have to drop it and re-add it. However, this screws with the existing constraints and it's just simpler to rebuild the table